### PR TITLE
Added WriteBatch block to simple_example.cc

### PR DESCRIPTION
**Pull request by alabid**

### CODE REVIEW

File: examples/.gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 column_families_example
 simple_example
+c_simple_example
+compact_files_example
```
Comment (igorcanadi) [inline examples/.gitignore]: Can you pls add a newline here?

**Comment by igorcanadi** [comment]
lgtm, just one small comment.

File: examples/.gitignore (hunk 1 above)
Comment (alabid) [inline examples/.gitignore]: Done.

**Comment by igorcanadi** [comment]
Tnx!

**Pull request merged**